gax/src/test/java/com/google/api/gax/rpc/BatchedRequestIssuerTest.java

```diff
@@ -29,28 +29,25 @@
  */
 package com.google.api.gax.rpc;
 
-import com.google.common.truth.Truth;
+import static com.google.common.truth.Truth.assertThat;
```

Comment:
Don't static import this since there are at least three different assertThat's this could be